### ISSUE TITLE
Unable to build on Ubuntu 16.04 32-bit

### ISSUE THREAD

**Issue by igor2x** [ebruck/radiotray-ng#90]
Hi,
on Ubuntu 16.04 Desktop **32-bit** I downloaded radiotray-ng-0.2.4.tar.gz and then followed instructions "To Build on Ubuntu" from https://github.com/ebruck/radiotray-ng and at command:
`cmake .. -DCMAKE_BUILD_TYPE=Release`

I got error:

> CMake Error at CMakeLists.txt:1 (cmake_minimum_required):
>   CMake 3.10 or higher is required.  You are running version 3.5.1
> -- Configuring incomplete, errors occurred!
> 

It looks like when cmake is installed from Ubuntu 16.04 it installs older version then radiotray-ng requires in the latest source code.

**Comment by igor2x** [comment]
In order to compile radiotray-ng I need to install newer version of cmake.

I looked at official cmake web page https://cmake.org/download/ and there is no cmake binary for Linux 32-bit. It is getting more and more interesting...

I solved the problem with the steps bellow.

1. I removed cmake:
`sudo apt purge cmake`

2. I downloaded cmake-3.13.2.tar.gz source code from above URL and compiled and installed cmake myself:
```
tar xf cmake-3.13.2.tar.gz
cd cmake-3.13.2
./configure
make
sudo make install
```

3. Close down a terminal and open new terminal.
Continue with instructions from radiotray-ng build site with:
`cmake .. -DCMAKE_BUILD_TYPE=Release`
etc. to the end of command list without a problem.

4. I have launched radiotray-ng and from About I see v2.0.4. It looks like I succeeded.

It is getting harder and harder to install radiotray-ng. There was a time 32-bit Ubuntu deb package was available. Not complaining, just saying.
